pages/filter.tsx

```diff
@@ -15,6 +15,15 @@ import { ButtonBase } from '../src/Components/ButtonBase'
 import { NotificationsContext } from '../src/context/NotificationsContext'
 import { VerticalSpacing } from '../src/Components/VerticalSpacing'
 
+const Heading = styled.section`
```

Comment:
I'm not sure this lines up with the area below - it might be easier to keep the h1 in that section rather than a section element for each.